sysutils/ioc/Makefile

```diff
@@ -23,6 +23,11 @@ GH_PROJECT=	ioc
 NO_ARCH=	yes
 NO_BUILD=   yes
 
+SHEBANG_LANG= python
+SHEBANG_FILES= ./bin/ioc
+python_OLD_CMD= "/usr/local/bin/python3.6"
```

Comment:
This should be python3.7, as the latest master branch of ioc now defaults to that.

Reply:
this is the *old* command.